doc/man/8/ceph.rst

```diff
@@ -580,7 +580,7 @@ Subcommand ``module ls`` will list currently enabled manager modules (plugins).
 
 Usage::
 
-  ceph mgr module ls
+  ceph mgr module ls {detail}
```

Comment:
Is the suggestion here that `ceph mgr module ls` (or `ceph mgr module ls detail` if you like this implementation) should go one step further and print the data in a `prettytables` format?

Reply:
@ljflores since this is not Python, it's tougher to get this fancy output, so I'm fine with the plain-text output. My main comment was about Ceph CLI defaulting to plain-text output unless `--format=json` is used (I always found weird to get json output in some Ceph CLI commands).

If we want another compact output, what about this?:
```
ceph mgr module ls
alerts:     -
balancer:   on (always_on)
crash:      on (always_on)
dashboad:   on

```
I was also going to suggest exploring unicode (✔️ and ❌ ) and colors, but you never know how that will play with terminals/shells/etc....

Reply:
Oh I see what you mean. I'll look for a "pretty-print" option in Boost. If there isn't one (I'm not sure if Boost offers something like Python's `prettytables` or not), then I'll see about doing it manually.